test/e2e/bm.e2e.test.ts

```diff
@@ -1,86 +1,90 @@
 import * as mwoffliner from '../../src/mwoffliner.lib.js'
 import { execa } from 'execa'
+import { RENDERERS_LIST } from '../../src/util/const.js'
 import rimraf from 'rimraf'
-import { zimcheckAvailable, zimcheck } from '../util.js'
+import { zimcheckAvailable, zimdumpAvailable, zimcheck, zimdump } from '../util.js'
 import 'dotenv/config.js'
 import { jest } from '@jest/globals'
-import { zimdumpAvailable, zimdump } from '../util.js'
 
-jest.setTimeout(200000)
+jest.setTimeout(800000)
 
 describe('bm', () => {
-  const now = new Date()
-  const testId = `mwo-test-${+now}`
-
-  const parameters = {
-    mwUrl: 'https://bm.wikipedia.org',
-    adminEmail: 'test@kiwix.org',
-    outputDirectory: testId,
-    redis: process.env.REDIS,
-    format: ['nopic'],
-  }
+  for (const renderer of RENDERERS_LIST) {
```

Comment:
All of this will come in each e2e test, why not something like `testWithAllRenderers([options like "format" or  mwUrl"], {callback code containing only the test})`

With variables like `outFiles` or `Dump`.

We should have a test with the only the very unique specific part.

Here I see far too much duplicate code